rts/lra/coordinator/src/test/java/io/narayana/lra/coordinator/tools/osb/mbean/ObjStoreBrowserLRATest.java

```diff
@@ -5,7 +5,6 @@
 
 package io.narayana.lra.coordinator.tools.osb.mbean;
 
-import com.arjuna.ats.arjuna.common.Uid;
```

Comment:
I think the title of the pull request should be "Fix Javadoc and optimize import". I think this commit should be reworded to "Remove unused import". Is that OK with you?

Reply:
Thanks @tomjenkinson , what about reword the commit to "Remove unused import and zip file"?

Reply:
That works for me, thank you